apps/condo/domains/user/schema/ConfirmPhoneAction.js

```diff
@@ -196,14 +283,18 @@ const ConfirmPhoneActionService = new GQLCustomSchema('ConfirmPhoneActionService
                     },
                     sender: sender,
                 })
-                return 'ok'
+                return { status: 'ok' }
             },
         },
         {
             access: true,
-            schema: 'completeConfirmPhoneAction(token: String!, smsCode: Int!): String',
+            schema: 'completeConfirmPhoneAction(data: CompleteConfirmPhoneActionInput!): CompleteConfirmPhoneActionOutput',
             resolver: async (parent, args, context, info, extra) => {
-                const { token, smsCode } = args
+                const { token, smsCode, captcha } = args.data
+                const { error } = await captchaCheck(captcha, 'complete_verify_phone')
+                if (error) {
+                    throw new Error(`${CAPTCHA_CHECK_FAILED}] ${error}`)
+                } 
```

Comment:
it's better also to use the lock in the complete mutation too.

Reply:
Done (small lock for 5 seconds against brute force attacks or if the user failed with our `auto send input`)
Anyway,  we have limited attempts on complete